### PR TITLE
Take copied modulemd_defaults into consideration for module errata copy

### PR DESCRIPTION
**Pull request by dralley**
re: #5055
https://pulp.plan.io/issues/5055

Dep: https://github.com/pulp/pulp_rpm/pull/1400

### CODE REVIEW

**Comment by kersommoura** [comment]
@dralley, thank you for update the tests.

**Comment by kersommoura** [comment]
Is there a specific version of Pulp that this change will be effective?

I am assuming any version after 2.20. We should add a logic to assure that this test does not fail in a earlier version of Pulp. Let me find an example.

**Comment by kersommoura** [comment]
A minor suggestion is to update the docstring with the number of issue related to this change. This can be very helpful later on.

**Comment by bherrin3** [comment]
Oh, and +1 @dralley for making these changes! Awesome job!

**Comment by dralley** [comment]
@kersommoura @bherrin3 Updated

File: pulp_2_tests/tests/rpm/api_v2/test_modularity.py [hunk 1]
```diff
@@ -1450,7 +1451,7 @@ class ModularErrataCopyTestCase(unittest.TestCase):
     def setUpClass(cls):
         """Create class-wide variables."""
         cls.cfg = config.get_config()
-        if cls.cfg.pulp_version < Version('2.19'):
+        if cls.cfg.pulp_version < Version('2.21'):
```
Comment (bherrin3) [inline pulp_2_tests/tests/rpm/api_v2/test_modularity.py]: The following line needs to be updated as well
```
           raise unittest.SkipTest('This test requires Pulp 2.21 or newer.')
```

**Comment by bherrin3** [comment]
Ack a conversation with @dralley as he feels the parent PR (https://github.com/pulp/pulp_rpm/pull/1400) will be merged shortly. 

Else we could update with the following, if there is a source Redmine Issue for `XXXX` and merge now.:

```
        if cls.cfg.pulp_version < Version('2.21'):
            raise unittest.SkipTest('This test requires Pulp 2.21 or newer.')
            if not selectors.bug_is_fixed(XXXX, cls.cfg.pulp_version):
                raise unittest.SkipTest('https://pulp.plan.io/issues/XXXX')
```

**Comment by dralley** [comment]
1400 merged

**Comment by kersommoura** [comment]
@dralley, thanks.

**Pull request merged**